server/database/queries.js

```diff
@@ -4,32 +4,208 @@ const db = require('./database.js');
+  const queryStr = 'SELECT * from rules WHERE id = ?';
+  query(queryStr, [id], callback);
+};
 
-  getHouseAddress(id, callback) {
-    const queryStr = 'SELECT * from addresses a INNER JOIN full_listing fl ON fl.name_id = ? AND a.id = fl.addresses_id';
-    query(queryStr, [id], callback);
-  },
+const createHouseRules = (args, callback) => {
+  const queryStr = 'INSERT INTO rules (check_in_start, check_in_end, check_out, kid_friendly, credit_cards, age_restriction, curfew, lock_out, non_smoking, pet_friendly, taxes_included, cancellation, important_notes_one, important_notes_two, important_notes_three, important_notes_four, important_notes_five) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)';
+  const newArgs = [
+    args.check_in_start,
+    args.check_in_end,
+    args.check_out,
+    args.kid_friendly,
+    args.credit_cards,
+    args.age_restriction,
+    args.curfew,
+    args.lock_out,
+    args.non_smoking,
+    args.pet_friendly,
+    args.taxes_included,
+    args.cancellation,
+    args.important_notes_one,
+    args.important_notes_two,
+    args.important_notes_three,
+    args.important_notes_four,
+    args.important_notes_five,
+  ];
+  query(queryStr, newArgs, (err, data) => {
+    if (err) {
+      callback(err, null);
+    } else {
+      getHouseRulesByID(data.insertId, callback);
+    }
+  });
+};
+
+const updateHouseRules = (args, callback) => {
+  const queryStr = 'UPDATE rules SET check_in_start = ?, check_in_end = ?, check_out = ?, kid_friendly = ?, credit_cards = ?, age_restriction = ?, curfew = ?, lock_out = ?, non_smoking = ?, pet_friendly = ?,taxes_included = ?, cancellation = ?, important_notes_one = ?, important_notes_two = ?, important_notes_three = ?, important_notes_four = ?, important_notes_five = ? WHERE (id = ?)';
+  const newArgs = [
+    args.check_in_start,
+    args.check_in_end,
+    args.check_out,
+    args.kid_friendly,
+    args.credit_cards,
+    args.age_restriction,
+    args.curfew,
+    args.lock_out,
+    args.non_smoking,
+    args.pet_friendly,
+    args.taxes_included,
+    args.cancellation,
+    args.important_notes_one,
+    args.important_notes_two,
+    args.important_notes_three,
+    args.important_notes_four,
+    args.important_notes_five,
+    args.id,
+  ];
+  query(queryStr, newArgs, (err) => {
+    if (err) {
+      callback(err, null);
+    } else {
+      getHouseRulesByID(args.id, callback);
+    }
+  });
+};
 
-  getHouseDescription(id, callback) {
-    const queryStr = 'SELECT * from descriptions d INNER JOIN full_listing fl ON fl.name_id = ? AND d.id = fl.descriptions_id';
-    query(queryStr, [id], callback);
-  },
+const deleteHouseRules = (id, callback) => {
+  const queryStr = 'DELETE FROM rules WHERE (id = ?)';
+  query(queryStr, [id], (err, data) => {
+    if (err) {
+      callback(err, null);
+    } else {
+      callback(null, data);
+    }
+  });
 };
+
+module.exports.getHouseInfoByHostelID = getHouseInfoByHostelID;
```

Comment:
Not a huge deal, but could you clean up these lines with one module.exports object?

i.e. module.exports = {
     getHouseInfoByHostelID,
     getHouseRules, etc....
}

Not a big deal if you're never going to use this script again...

Reply:
Done! Thank you for reminding me!